src/pages/challenges/index.tsx

```diff
@@ -12,6 +12,12 @@ import { skipToken } from '@reduxjs/toolkit/dist/query';
 import useAuthenticated from '@/hooks/useAuthenticated';
 import { useGetChallengesQuery } from '@/app/services/challengesApi';
 
+export const loginNode = (
+    <a href={LOGIN_URL} target="_blank" rel="noreferrer">
+        Click here to Login
+    </a>
+);
+
```

Comment:
if this is being used in other files also, please take it out into a separate file and make it <LoginButton/>